lib/ansiblelint/skip_utils.py

```diff
@@ -26,7 +26,7 @@
 import ruamel.yaml
 from typing import Any, Generator, List, Sequence
 if sys.version_info >= (3, 8):
-    from typing import Literal
+    from typing import Literal  # pylint: disable-msg=E0611
```

Comment:
I think the proper fix would be try-except style

Reply:
@webknjaz This is not possible in this case, is mostly a bug in pylint which fails because it goes inside this branch with an older version of python. I found that online. 

Python itself will not fail with this import, only pylint would complain.

Reply:
Well, I have to agree with @webknjaz here. Looks like the proper fix would probably be to use a try-except, or would pylint complain about that?